resources/user-agents/browsers/misc/web-explorer.json

```diff
@@ -0,0 +1,28 @@
+{
+  "division": "Web Explorer",
+  "sortIndex": 950,
+  "lite": false,
+  "standard": true,
+  "userAgents": [
+    {
+      "userAgent": "Web Explorer",
+      "browser": "web explorer",
+      "platform": "Android",
+      "engine": "Blink",
+      "device": "general Mobile Phone (Touch)",
+      "properties": {
+        "Parent": "DefaultProperties",
+        "Comment": "Web Explorer"
+      },
+      "children": [
+        {
+          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) Version/* Web Explorer/* Chrome/* Mobile Safari*",
+          "platforms": [
+            "Android_9_0", "Android_9_0_B",
```

Comment:
issue-2101-BG

Reply:
This new `_B` platform is for cases like this `... (Linux; Android 9) ...`, when there is nothing between the `9` and the `)`. The normal rule requires a semicolon after the `9`. Do you have any idea to allow these two cases only without allowing the `9.0` fakes too?

Reply:
Ah, yeah, I see, I don't think there's a way to do that with the Browscap format since you can't use `?` in place of the semi-colon, as it'll require that a character be there (there's no regex `?` equivalent in the browscap format).

Does Web Explorer use both formats?

Reply:
I only the `_B` version, I removed the normal version.